myst_nbs/case_studies/moderation_analysis.myst.md

```diff
@@ -6,27 +6,32 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: pymc-dev-py39
   language: python
-  name: python3
+  name: pymc-dev-py39
 ---
 
+(moderation_analysis)=
 # Bayesian moderation analysis
 
-**Author:** [Benjamin T. Vincent](https://github.com/drbenvincent)
+:::{post} March, 2022
+:tags: moderation, path analysis,
+:category: beginner
```

Comment:
```suggestion
:category: beginner, explanation
```

I think this notebook fits relatively well the explanation concept, especially now that auxiliary plotting is hidden.